runbotics/runbotics-desktop/src/core/bpm/CamundaExtension.js

```diff
@@ -6,6 +6,8 @@ export default {
     moddleOptions,
 };
 
+const outputVariables = {};
```

Comment:
this approach can lead to sharing variables between different process executions. Propose a way to handle it within a single process execution but remember that we can have subprocesses